_includes/quickstart.markdown

```diff
@@ -18,7 +18,18 @@ If you have any trouble with the process, or have ideas about how to improve thi
 
 Note: This alpha release is for Mac OS X and 64-bit Linux only. Windows users, there will be a Unison release for you some day soon.
 
-#### Option 1: Using Homebrew
+#### Option 1: Using Nix (OSX & linux)
+
+First [install nix]: https://nixos.org/nix/manual/#ch-installing-binary if you haven't already.
+
+Then, from the command line, enter or paste this command:
+```
+nix-env -i unison-ucm
```

Comment:
This is going to depend on which channel you’re using, right? Currently I think it’s only on master. I have an overlay defined for master, so I can do `nix-env -i master.unison-ucm`, but it won’t be that simple for most.

I wonder if this change should wait until Unison is in a more widespread release, or if it should at least call out which channels have it.